App/components/RadioButton/RadioButton.styled.ts

```diff
@@ -9,3 +10,25 @@ export const RadioButtonContainer = styled.View`
 export const RadioButtonText = styled.Text`
   color: black;
 `
+
+export const RadioButtonViewUnfilled = styled.View`
```

Comment:
Why we can't use only one Radio Button View with rather styles depends on props?
As far as I can judge this is a duplication of code)